### ISSUE TITLE
AttributeError in module pyabf.tools when trying to use Memtest

### ISSUE THREAD

**Issue by konung-yaropolk** [swharden/pyABF#118]
When trying to run Memtest the same way as in the guide, the next error occurs.
A few months ago there aren't any issues when running this code.

`abf = pyabf.ABF('file.abf')`
 `memtest = pyabf.tools.memtest.Memtest(abf)`

  File "/home/lol/abfplot/src/abfplot_core.py", line 215, in membrane_test
    memtest = pyabf.tools.memtest.Memtest(abf)
  File "/home/lol/.local/lib/python3.9/site-packages/pyabf/tools/memtest.py", line 27, in __init__
    Result = pyabf.tools.sweep.SweepMeasurement
AttributeError: module 'pyabf.tools' has no attribute 'sweep'

**Comment by konung-yaropolk** [comment]
I have solved the issue by adding the necessary imports to the memtest.py
A pull request is waiting for you :)

**Comment by swharden** [comment]
This has been fixed and will be released on pypi shortly. Thanks @konung-yaropolk for pointing it out and creating the PR!

For the record I intend to keep pyabf focused on _reading_ ABF files, and users are highly encouraged to write their own analysis software. The `pyabf.tools.memtest` module works but it's not optimized for efficiency or accuracy. I'll update the documentation website to make this more clear.

Users interested in learning more about membrane test calculations are encouraged to check-out [**Exploring the Membrane Test with a Voltage-Clamped Neuron Model**](https://swharden.com/blog/2020-10-11-model-neuron-ltspice) (a blog post I wrote last year that does a deep dive into membrane test calculations)

[![](https://user-images.githubusercontent.com/4165489/125146146-73890880-e0f2-11eb-9994-c7c5ab183599.png)](https://swharden.com/blog/2020-10-11-model-neuron-ltspice)
